Common subdirectories: xcb-proto-1.14.1.orig/doc and xcb-proto-1.14.1/doc
Common subdirectories: xcb-proto-1.14.1.orig/src and xcb-proto-1.14.1/src
diff -u xcb-proto-1.14.1.orig/xcb-proto.pc.in xcb-proto-1.14.1/xcb-proto.pc.in
--- xcb-proto-1.14.1.orig/xcb-proto.pc.in	2020-10-08 22:31:54.000000000 +0000
+++ xcb-proto-1.14.1/xcb-proto.pc.in	2022-04-05 15:59:21.958915952 +0000
@@ -4,6 +4,7 @@
 datadir=@datadir@
 libdir=@libdir@
 xcbincludedir=${pc_sysrootdir}@xcbincludedir@
+PYTHON_PREFIX=@PYTHON_PREFIX@
 pythondir=${pc_sysrootdir}@pythondir@
 
 Name: XCB Proto
Common subdirectories: xcb-proto-1.14.1.orig/xcbgen and xcb-proto-1.14.1/xcbgen
